### PR TITLE
Loading and header

### PR DESCRIPTION
**Pull request by JCraigWasTaken**

### CODE REVIEW

File: dev-frontend/src/comps/matchupList.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ class MatchupList extends React.Component{
       // Need to create some kind of "Loading" component
```
Comment (topherbentz) [inline dev-frontend/src/comps/matchupList.js]: You can remove this comment now.

File: dev-frontend/src/pageComps/homePage.js [hunk 2]
```diff
@@ -80,28 +82,95 @@ class HomePage extends React.Component{
...
+                        className='searchBar'/>
+                      <div
+                        id='filterBar'
+                        className='flex-row filterBar'>
+                        <div
+                        className='filterDiv'
+                          style={{'flex':1}}>
+                          <span>Filter</span>
+                        </div>
+                        <select
```
Comment (topherbentz) [inline dev-frontend/src/pageComps/homePage.js]: All of these options will need to come from the backend eventually right?

Comment (JCraigWasTaken) [reply dev-frontend/src/pageComps/homePage.js]: Yeah all of the selects will probably come from the back-end.

File: dev-frontend/src/comps/matchupList.js (hunk 1 above)
Comment (JCraigWasTaken) [reply dev-frontend/src/comps/matchupList.js]: I just removed it, we should be good

**Comment by JCraigWasTaken** [comment]
I think I responded to all the comments

**Pull request merged**